qa/suites/upgrade/octopus-x/parallel/0-cluster/start.yaml

```diff
@@ -0,0 +1,40 @@
+meta:
```

Comment:
looks like we can't use the same cluster as  https://github.com/ceph/ceph/blob/master/qa/suites/rados/cephadm/upgrade/fixed-2.yaml , right?